examples/FileHandling/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# CMake file for DISTRHO Plugins #
+# ------------------------------ #
+
+dpf_add_plugin(d_files
+  TARGETS jack lv2 vst
+  UI_TYPE opengl
```

Comment:
UI_TYPE should default to opengl if not specified. is this possible?

Reply:
When `UI_TYPE` is not given, it designates a plugin without UI.
How else do you suggest to describe a non-UI plugin?

Reply:
By having the list of UI files to build empty.
Current DPF makefiles already do this. the UI_TYPE is only needed when it is != opengl.

https://github.com/DISTRHO/DPF/blob/master/Makefile.plugins.mk#L73